src/components/funFacts/FunFactComponent.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react"
+import { funFacts, FunFact } from "./funfacts"
+import { useState } from "react"
+import "./FunFactPage.scss"
+
+interface FunFactsProps {
+    funFacts: FunFact[];
+}
+
+export const FunFactComponent: React.FunctionComponent<FunFactsProps> = ({ funFacts }) => {
+
+  const [selectedFact, setSelectedFact] = useState<number>(0);
+
+  function toggleFactDetail(funFactId:number){
+    if (selectedFact===funFactId){
+      setSelectedFact(0);
+    }
+    else {
+      setSelectedFact(funFactId);
+    }
+  }
+
+  return(
+    <div>
+      {funFacts.map( funFact => (
+        <div className="funfact">
+          <div className="funfact__prompt">
+            <h2 className="funfact__prompt--question">{ funFact.prompt }</h2>
+            <button className={`funfact__button ${funFact.id===selectedFact ? "funfact__button--revealed" : ""}`} onClick={ () => toggleFactDetail(funFact.id)}>^</button>
+          </div>
+            {funFact.id===selectedFact? <p className="funfact__detail">{ funFact.detail }</p> : null }
+        </div>
+        )
+        )
```

Comment:
I'm not sure of the correct indenting, but you shouldn't end up with two brackets like this!

Reply:
I'm not sure either, their parent parentheses are on the same line (24) so i will put these on the same line